### ISSUE TITLE
Pitchbend and Modwheel support

### ISSUE THREAD

**Issue by EthanSK** [flit/MidiKeys#33]
Would be cool to support pitchbend and modwheel. just a simple bar that shows the value.

**Comment by flit** [comment]
Agreed—there's a lot of cool stuff that could be added to MidiKeys for sure. But lately it's mostly up to other users to contribute changes, since I don't use it myself much these days and have other projects that consume all my time.

**Comment by EthanSK** [comment]
wish it was in swift :/

**Comment by flit** [comment]
That just shows how old most of the MidiKeys code is…
